packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapsPlugin.java

```diff
@@ -94,5 +100,34 @@ public void onActivityDestroyed(Activity activity) {
 
   private GoogleMapsPlugin(Registrar registrar) {
     this.registrarActivityHashCode = registrar.activity().hashCode();
+    if (dummyMapInitialized) {
+      return;
+    }
+    Activity activity = registrar.activity();
+    if (activity == null) {
```

Comment:
Looks like after first line of this method activity can't be null

Reply:
![image](https://user-images.githubusercontent.com/4225279/73026045-c4933700-3e41-11ea-85f2-662053229c8e.png)

Reply:
Yes. But if it will be null, crash occurs on line 102.

Reply:
You are right. Fixed